appveyor.yml

```diff
@@ -78,7 +68,8 @@ install:
   - sed -i "s/scipy==.*/scipy==%SCIPY_VERSION%/" etc/requirements.txt
 
   - conda info -a
-  - conda install conda=4.1.11 conda-build=1.21.11 anaconda-client=1.5.1 --yes -q
+  - conda install conda=4.3.30 conda-build=3.0.28 anaconda-client=1.6.3 --yes -q
```

Comment:
Are there any many main differences/reasons for using these newer versions, aside from fixing the build?

Reply:
There are definitely new features and bug fixes. Many are optional, like the new build strings and changes in how one pins dependencies (like the numpy C-API), which I haven't tried out. You noticed the consolidated recipes - I regenerated a bunch of ours.

The reason I updated in this PR was to fix our toolchain, in that the old miniconda could no longer work with the package index or update to newer conda versions.